hmvec/hmvec.py

```diff
@@ -387,6 +470,7 @@ def add_hod(self,name,mthresh=None,ngal=None,corr="max",
         self.hods[name]['bg'] = self.get_bg(Ncs,Nss,self.hods[name]['ngal'])
         self.hods[name]['satellite_profile'] = satellite_profile_name
         self.hods[name]['central_profile'] = central_profile_name
+        self.hods[name]['mthres'] = np.log10(mthresh[:,None])
```

Comment:
Please name it log10mthresh to avoid confusion in code.

Reply:
Done.